docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -553,8 +557,92 @@ as follows:
 	# Frozen normal distribution for MD, representing uncertainty in our effect size
 	bw_md_distribution = norm(mean, std)
 
-Determining Whether A Simulant is Affected
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Determining How A Simulant is Affected
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Hemoglobin Level**
+
+For the purposes of our simulation, we assumed that simulants only received an 
+effect from iron fortification if they directly consumed iron fortified foods 
+and that they received no effect of iron fortification on their hemoglobin 
+levels from maternal consumption of iron fortified foods.
+
+Additionally, we assumed that simulants in  begin to eat staple foods as a 
+supplement to breast milk consumption at the age of six months and that the 
+quantity of staple foods consumed as a proportion of total consumption 
+increases linearly from six months to two years of age, at which point it 
+reaches its peak and then remains constant.
+
+Further, we assumed that the full effect of the iron fortification 
+intervention takes six months to achieve and that the effect scales up in a 
+linear fashion between the onset of exposure and six months post exposure (
+this is likely a conservative assumption and the shape likely follows more of 
+a logarithmic curve). Additionally, we assumed that all individuals covered by 
```

Comment:
logistic curve?

Reply:
I wasn't sure what to call it, but this is the general shape (bounded exponential?). It didn't look logistic to me in that there was no lag at the beginning until the slope increased.

![image](https://user-images.githubusercontent.com/54918472/77488018-3f9cad80-6df1-11ea-9ed6-8cf042bad715.png)

Reply:
Perhaps just describe the shape of the curve more explicitly. For example:

"This is likely a conservative assumption, as there is evidence that the true curve increases more steeply at first, then levels off before reaching the full effect at six months. Thus, the measured response curve is concave down, and our linear approximation lies entirely below this curve -- it is the secant line to the curve, with slope equal to the average rate of increase over the 6 month interval."

Also, we should be sure to cite the source of this graph.

Reply:
Yes, I like your description, Nathaniel - I will update the language as such. And yeah, it's from the Andersson paper! I will include the reference.